guides/user/managed-delivery/intergate-your-CI/index.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: single
+title:  "Integrate your CI"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+One of Managed Delivery goals is to be able to show a code journey, from committing it until it gets deployed,
+in an easy, consistent way.
+In order to enable it, we created an integration between MD and the CI provider, which described below.
+
+## Publishing a new artifact
+
+When a new artifact is published, `keel` gets notified by `echo` (for debians) or by `igor` (for dockers). Each artifact has a `metadata` section, which is controlled by the artifact supplier. 
+For example, for `docker` artifacts, the `metadata` field is being populated in `clouddriver` based on the docker registry. 
+
+Here is an example of a docker artifact event, that `keel` is tracking:
```

Comment:
Suggestions:
```suggestion
When a new artifact is published, `keel` gets notified by `echo` (for Debian packages) or by `igor` (for Docker images). Each artifact has a `metadata` section, which is populated by the corresponding artifact supplier implementation in `keel`. 
For example, for `docker` artifacts, the `metadata` field is populated by querying `clouddriver`, which stores information from the docker registries configured in the Spinnaker installation. 

Here is an example of a Docker artifact event tracked by `keel`:
```

Reply:
@gal-yardeni Who's the target audience for this doc? I originally assumed it was users, but this is going deep into the "nitty gritty" of the implementation...

Reply:
@luispollo It's for users who wish to use the functionality we are offering. I wanted to explain the details of what we are doing so they can do the same if they wish. The aim of this doc is to explain the overview, plus how they can actually use it. Makes sense?

Reply:
Kinda. As I've tried to explain in other comments, I think the definition of a "user" in the Spinnaker docs is someone who just interacts with the Spinnaker UI or CLI to use delivery and infrastructure management features, not someone who's concerned with the implementation details behind Spinnaker. We have other sections in the docs for developers, who'd be interested in the implementation details (`/guides/developer`) and operators , who'd be interested in how to configure the integration in their own installation of Spinnaker (`/guides/operator`). My suggestion is to split this content into specific pages for those specific audiences.

Reply:
Thanks Luis! what you are saying makes sense to me, but when I'm looking at the current spinnaker.io website I don't see any other guide splitting up between sections 🤔 . 
I don't know what is the right call here -- if you are looking from the `user` lense as you mentioned, they can't really use it as is as this whole experience is applicable for Netflix only at this point. I'm not sure where is the right place to locate this guide. Curious to hear other thoughts maybe -- @emjburns / @erikmunson?

Reply:
After thinking about it for a bit, I'll split it between a user guide and a developer guide with cross-reference between them. will update the PR soon!